#goal station
string fiducial_id
---
#result
string result
